spec/lago/api/connection_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Lago::Api::Connection do
+  subject(:connection) do
+    described_class.new(
+      'fake-api-key',
+      URI('https://testapi.example.org/')
+    )
+  end
+
+  context "when an unsuccessful request is made" do
+    before do
+      stub_request(:get, 'https://testapi.example.org/NOTFOUND')
+        .to_return(
+          body: JSON.generate({status: 404, error: "Not Found"}),
+          status: 404)
+    end
+
+    it "raises an exception with an integer error code" do
+      expect { connection.get('/NOTFOUND') }.to raise_error { |exception|
```

Comment:
connection class does not have `get` method. We can maybe use `post` for testing purposes...

Reply:
@lovrocolic I actually added `get` in #16 which is how I found this issue, guess I had already merged it in my fork.  But will update so the tests pass as is.